admin/scss/_SecurityAdmin.scss

```diff
@@ -41,6 +41,9 @@ green tick icon as a background this is created using compass generated classes
 }
 
 .cms .cms-content .SecurityAdmin { //datagrid overflow on Security Admin
+	.add-new-button.font-icon-plus:before {
+    	content: "H";
```

Comment:
Instead of `content: "H";` it would be better to `@extend .font-icon-torso;` just in case the content value for that icon ever changes.